### ISSUE TITLE
Marketing Pages should be decoupled from core application

### ISSUE THREAD

**Issue by omenking** [forem/forem#2377]
**Is your feature request related to a problem? Please describe.**

So today the web-server is down which is a thing that can happen, but it's taking down the static pages as well. 

**Describe the solution you'd like**

I would suggest that these pages be decoupled from the Rails app and be separately served through a static website hosting.

The static pages could remain inside the repo.

![Screen Shot 2019-04-11 at 11 13 28 AM](https://user-images.githubusercontent.com/7776/55968931-f2a8e100-5c4a-11e9-97db-2ebcff801155.png)

I would suggest static website hosting with S3. Of course, I am biased with AWS. But regardless of proposed service to host, I think these pages should not go down with the server.

**Describe alternatives you've considered**
Maybe a DO solution would be more inline since they are an official sponsor.

**Additional context**
It's very easy to add a status page with status lambda
https://github.com/ks888/LambStatus

I'm quite certain this would cost nothing on AWS free tier as I run it on my own web-app ExamPro.

**Comment by aspittel** [comment]
Hey @omenking, that's a really cool idea! We'll discuss as a team and then get back to you. Thanks!

**Comment by metaskills** [comment]
Hi, my name is Ken Collins and on top of working at Custom Ink I spent a lot of my timing helping people learn AWS Lambda, specifically with Ruby/Rails. At Custom Ink we have been breaking up our monolith Rails app for years. Not for the sake of micro services but logically where they need to be optimized or segmented. 

To that end, I have created some simple ways (https://lamby.custominktech.com) to use Rails and Lambda. Is there any way I can work with the Dev.to team on this idea for y'all in some places?
